blocks/section/__intro/_place/section__intro_place_photo-editor.css

```diff
@@ -0,0 +1,16 @@
+.section__intro_place_photo-editor{
+  max-width: 422px; 
+  margin: 67px 0 75px;
+}
+@media screen and (max-width: 1199px) {
+  .section__intro_place_photo-editor{
+    max-width: 422px;
```

Comment:
I don't think it is necessary to write `max-width: 422px;` here because you have already written the same code on line 2. There is no need to write the same code in media queries.

Reply:
@MarPostovik it's done